### DISCUSSION TITLE
Force sync to the Git repo?

### DISCUSSION THREAD

**Discussion by johnnymatthews** [GitbookIO/community#285] [0. General bugs and questions]
Is there anyway to force GitBook to check the remote Git repo? I pushed changes to my remote repo 14 minutes ago, but GitBook hasn't realized yet. I pushed to the same branch that GitBook should be building from.

**Comment by johnnymatthews** [comment]
Nevermind. I delete the space and made a new one. That seemed to solve the problem for me.

**Reply by strazto** [reply]
Although it's good that you got around your problem, I don't think having to delete and re-init your space is a good solution, nor viable at scale. This should be re-opened
